README.adoc

```diff
@@ -145,306 +136,184 @@ The [hotspot=from file=0]`FROM` instruction initializes a new build stage and in
-7f24bdb73d53: Pull complete 
-264aaa453271: Pull complete 
-b095666e9645: Pull complete 
-6d080a23bd35: Pull complete 
-7bdfdc073f3b: Pull complete 
-7cf1f897277a: Pull complete 
-38693ac81335: Pull complete 
-1207f8b498f1: Pull complete 
-531978dfef2c: Pull complete 
-b67203a348c6: Pull complete 
-d0386355392b: Pull complete 
-Digest: sha256:676f0137e439e42b8f3250060a1ec2c4b9ae0ca851a4cb3cb2683256add838ec
-Status: Downloaded newer image for openliberty/open-liberty:kernel-java8-openj9-ubi
-docker.io/openliberty/open-liberty:kernel-java8-openj9-ubi
+************************************************************************
+*    Liberty is running in dev mode.
 ----
 
-Next, to build your image, make sure that your Docker daemon is running and execute the Docker `build`
-command from the active command-line session. If you execute your build from the same directory as your Dockerfile,
-you can use the period character (`.`) notation to specify the location for the build context. 
-Otherwise, use the `-f` flag to point to your Dockerfile:
+Open another command-line session and run the following command to make sure that your
+container is running and didn’t crash:
 
 [role='command']
 ```
-docker build -t ol-runtime .
+docker ps 
 ```
 
-Use the `-t` flag to give the image an optional name. In this case, `ol-runtime` is the name of your image.
-
-The first build usually takes much longer to complete than subsequent builds because Docker needs to
-download all dependencies that your image requires, including the parent image.
-
-If your build runs successfully, you'll see an output similar to the following:
+You should see something similar to the following output:
 
 [role="no_copy"]
 ----
-Sending build context to Docker daemon  316.5MB
-Step 1/6 : FROM openliberty/open-liberty:kernel-java8-openj9-ubi
- ---> c3cd510a72a2
-Step 2/6 : USER root
- ---> Running in a63a1bed8026
-Removing intermediate container a63a1bed8026
- ---> ace3b840f2fd
-Step 3/6 : RUN ln -s /opt/ol/wlp/usr/servers /servers
- ---> Running in 504b98ddff9a
-Removing intermediate container 504b98ddff9a
- ---> 1628ec9873b1
-Step 4/6 : USER 1001
- ---> Running in 1d8601e3896e
-Removing intermediate container 1d8601e3896e
- ---> a75cddf7cfc6
-Step 5/6 : ENTRYPOINT ["/opt/ol/wlp/bin/server", "run"]
- ---> Running in 231f5b20fb0c
-Removing intermediate container 231f5b20fb0c
- ---> 87501190599c
-Step 6/6 : CMD ["defaultServer"]
- ---> Running in 9377b2104726
-Removing intermediate container 9377b2104726
- ---> d8fd3f885f8f
-Successfully built d8fd3f885f8f
-Successfully tagged ol-runtime:latest
+CONTAINER ID        IMAGE                   COMMAND                  CREATED             STATUS              PORTS                                                                    NAMES
+ee2daf0b33e1        guide-docker-dev-mode   "/opt/ol/helpers/run…"   2 minutes ago       Up 2 minutes        0.0.0.0:7777->7777/tcp, 0.0.0.0:9080->9080/tcp, 0.0.0.0:9443->9443/tcp   liberty_dev
```

Comment:
Minor point but the last part of this output now shows as `liberty-dev` for the container name